src/site/stages/build/plugins/check-collections.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-console */
+const path = require('path');
+const collections = require('../data/collections.json');
+
+/**
+ * This checks whether there are broken collections in a front-matter .md file.
+ * It checks whether a collection or children is available in the MetalSmith collections list
+ * @param {object} buildOptions
+ */
+
```

Comment:
At the top of this file, can we give a complete example of how to configure a left nav similar to how we do with the [fragments](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/plugins/apply-fragments.js) middleware? I'd like to make this a standard for every build step.

Reply:
@ncksllvn can we create a separate ticket for this. I think we need to consolidate all the left rail nav stuff and then add the example in there. Right now let's just get this working so we can get some checks in while we are doing the CMS integration. I do agree that we need this somewhere close tot the code.

Reply:
Nah, no worries on the separate ticket. This is good as-is.